examples/quad/main.rs

```diff
@@ -442,6 +444,47 @@ fn main() {
         Backbuffer::Framebuffer(fbo) => (Vec::new(), vec![fbo]),
     };
 
+    let frames_in_flight = frame_images.len();
```

Comment:
It would be nicer to not strongly couple it with the swapchain size and make the decision on which element of the ringbuffers to use independent of the acquired frame id.

Reply:
As there are concerns regarding increased complexity. I think it would only require minor changes, but I'm fine with keeping it as it as well!